mongodb/service.sls

```diff
@@ -0,0 +1,8 @@
+start_mongodb_service:
+  service.running:
+    - running: True
+    - name: mongodb
+    - enable: True
+    - require:
+      - pkg: install_packages
+      - service: stop_mongodb_service
```

Comment:
When the service definition is in its own state file it shouldn't have explicit requires. Instead you should include this and use `require_in`. For example:
```saltstack
include:
  - .service

install_some_package:
  package.installed:
    - pkg: pkg_name
    - require_in:
        - service: start_mongodb_service
```